### PR TITLE
[CDAP-15674] toggle widget react migration

### PR DESCRIPTION
**Pull request by itsanudeep**
Toggle widget is now in react.
JIRA: https://issues.cask.co/browse/CDAP-15674
BUILD: https://builds.cask.co/browse/CDAP-UDUT371

### CODE REVIEW

File: cdap-ui/app/cdap/components/AbstractWidget/ToggleSwitchWidget/index.tsx [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+interface IWidgetAttributes {
+  on: IToggle;
+  off: IToggle;
+  default: string;
+}
+interface IToggleSwitchWidgetProps {
+  value: string;
+  config: IConfigType;
+  disabled: boolean;
+  onChange: (arg0: any) => void;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/AbstractWidget/ToggleSwitchWidget/index.tsx]: will `arg0` always be string?

File: cdap-ui/app/cdap/components/AbstractWidget/ToggleSwitchWidget/index.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  'widget-attributes': IWidgetAttributes;
+}
+interface IToggle {
+  label: string;
+  value: string;
+}
+interface IWidgetAttributes {
+  on: IToggle;
+  off: IToggle;
+  default: string;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/AbstractWidget/ToggleSwitchWidget/index.tsx]: Do we know if `default` will always be provided? Can we make it optional in implementation and fallback to `on` if provided?

**Review by ajainarayanan — COMMENTED** [review]
Minor comments.

**Comment by itsanudeep** [comment]
@ajainarayanan addressed the feedback, can you take another pass at this?

File: cdap-ui/app/cdap/components/AbstractWidget/ToggleSwitchWidget/index.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+  value: string;
+}
+interface IWidgetAttributes {
+  on: IToggle;
+  off: IToggle;
+  default?: string;
+}
+interface IToggleSwitchWidgetProps {
+  value: string;
+  config: IConfigType;
```
Comment (elfenheart) [inline cdap-ui/app/cdap/components/AbstractWidget/ToggleSwitchWidget/index.tsx]: be more specific with the props, don't pass the whole configs

**Comment by itsanudeep** [comment]
@elfenheart made changes based on the feedback

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Pull request merged**